core/src/main/java/org/apache/accumulo/core/client/sample/RowColumnSampler.java

```diff
@@ -80,8 +83,11 @@ private boolean hashField(SamplerConfiguration config, String field) {
   }
 
   @Override
-  protected boolean isValidOption(String option) {
-    return super.isValidOption(option) || VALID_OPTIONS.contains(option);
+  public void validateOptions(Map<String,String> config) {
+    super.validateOptions(config);
+    for (Map.Entry<String,String> entry : config.entrySet()) {
```

Comment:
Since you're only using the key, and not validating any values in this implementation, you could just loop over the `keySet()` rather than the `entrySet()`.

Reply:
Good catch